libs/elements/layout/footer/src/lib/components/footer/footer.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="footer">
+  <ng-content select="[footer]"></ng-content>
+  
+  <div class="footer-navigations">
+    <div class="headquarters">
+      <h4>Headquarters</h4>
+      <a>The Promenade, 19 General</a><br/>
+      <a>Mathenge Rd</a><br/>
+      <a>Nairobi, Kenya</a>
+    </div>
+    <div class="contact">
+      <h4>Contact</h4>
+      <a>T +254 78 92 27 755</a><br/>
+      <a>E info@elewa.co.ke</a>
+    </div>
+    <div class="activities">
+      <h4>Navigation</h4>
+      <a>About us</a><br/>
+      <a>Social impact</a><br/>
+      <a>Invest</a><br/>
+      <a>News</a><br/>
+      <a>Contact</a>
+    </div>
```

Comment:
Use angular router links to link to the given pages. 
Here is a nice tutorial that can guide you through this. https://angular.io/guide/router-tutorial-toh